xds/src/main/java/io/grpc/xds/ClusterImplLoadBalancerProvider.java

```diff
@@ -88,11 +88,22 @@ static final class ClusterImplConfig {
     // Drop configurations.
     final List<DropOverload> dropCategories;
     // Provides the direct child policy and its config.
-    final PolicySelection childPolicy;
+    final Object childConfig;
 
     ClusterImplConfig(String cluster, @Nullable String edsServiceName,
         @Nullable ServerInfo lrsServerInfo, @Nullable Long maxConcurrentRequests,
-        List<DropOverload> dropCategories, PolicySelection childPolicy,
+        List<DropOverload> dropCategories,
+        io.grpc.internal.ServiceConfigUtil.PolicySelection childPolicy,
```

Comment:
Would be nice to restore the import for PolicySelection

Reply:
Oh. This was for transitioning. The entire constructor was intended to be deleted. I've deleted it (after converting some callers in tests.)